File: TuristickaAgencija.Model/Uplate.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TuristickaAgencija.Model
{
    public class Uplate
    {

        public int Id { get; set; }
        public DateTime Datum { get; set; }
        public double Iznos { get; set; }
        public int RezervacijaId { get; set; }
        public int KorisnikId { get; set; }
    }
}

File: UI/turisticka_agencija_admin/lib/models/Uplate.dart
import 'package:json_annotation/json_annotation.dart';

part 'Uplate.g.dart';

@JsonSerializable()
class Uplate {
  String datum;
  int id;
  double iznos;
  int rezervacijaId;
  int korisnikId;

  Uplate(
      {required this.datum,
      required this.id,
      required this.iznos,
      required this.rezervacijaId,
      required this.korisnikId});

  factory Uplate.fromJson(Map<String, dynamic> json) => _$UplateFromJson(json);

  Map<String, dynamic> toJson() => _$UplateToJson(this);
}
